packages/generator-single-spa/src/vue/generator-single-spa-vue.js

```diff
@@ -4,8 +4,34 @@ const util = require("util");
 const commandExists = util.promisify(require("command-exists"));
 const chalk = require("chalk");
 const path = require("path");
+const fs = require("fs");
+const isValidName = require("../naming");
 
 module.exports = class SingleSpaVueGenerator extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+
+    this.option("orgName", {
+      type: String,
+    });
+  }
+  async getOptions() {
+    while (!this.options.orgName) {
+      let { orgName } = await this.prompt([
+        {
+          type: "input",
+          name: "orgName",
+          message: "Organization name (use lowercase and dashes)",
+        },
+      ]);
+
+      orgName = orgName && orgName.trim();
+      if (!orgName) console.log(chalk.red("orgName must be provided!"));
+      if (!isValidName(orgName))
+        console.log(chalk.red("orgName must use lowercase and dashes!"));
```

Comment:
Is this meant to be a console warning, or an error thrown? The equivalent if statement for projects throws an error.

Reply:
It's meant to be a console warning. It will display that line and prompt the user again until it passes validation. If it were to throw an error instead, the process would exit which I would think is undesirable since it can be handled gracefully.

Can you share an example of the equivalent if statement are you thinking of? I had changed the orgName and projectName prompts throughout to work as I described above in PR #145.

Reply:
https://github.com/single-spa/create-single-spa/pull/156/files#diff-4bf5d50eb100da1f8c558788d5ac79bdR55

Reply:
I see. That is a slightly different case because dir is an "option" not a "prompt". The Vue app create is sightly different than for the others because the `projectName` is derived from the `dir` option; the others prompt for the `projectName`. Since its not a prompt, an error is the only way to handle this for Vue apps. I'm open to changing it so that this generator prompts for the projectName instead.

Reply:
I see, makes sense 👍